modules/core/shared/src/main/scala/lucuma/core/model/ids.scala

```diff
@@ -11,4 +11,5 @@ object CallForProposals extends WithGid('c'.refined)
 object Group            extends WithGid('g'.refined)
 object Observation      extends WithGid('o'.refined)
 object Program          extends WithGid('p'.refined)
+object ProgramUser      extends WithGid('r'.refined)
```

Comment:
Hopefully I'm in the running for longest PR description for a one-line change.

Reply:
A quick search showed we're already using `r` for 'StandardRole` in Role.scala. Is that a problem?

Reply:
Yeah we're running out of letters but it would be best to choose another one.

Reply:
Alright, how do we feel about `m`?  The only other letter in `ProgramUser` we haven't used is `e`, I think, unless that is used for events or something.

Reply:
`m` is good. `e` is used for `ExecutionEvent`. The letters are starting to be non-mnemonic.